index.html

```diff
@@ -0,0 +1,171 @@
+            <div class="container_program">
+                <div class="row">
+                    <div class="col-md program_item">
+                        <i class="fa fa-cutlery fa-2x pb-2 pt-3" aria-hidden="true"></i>
+                        <h4 class="py-2">Pancake</h4>
+                        <p class="pt-2 pb-3">
+                            A pancake is a flat cake, often thin and round, prepared from a starch-based batter that may contain eggs, 
+                            milk and butter and cooked on a hot surface.
+                        </p>
+                    </div>
+                    <div class="col-md program_item">
+                        <i class="fa fa-birthday-cake fa-2x pb-2 pt-3" aria-hidden="true"></i>
+                        <h4 class="py-2">Pizza</h4>
+                        <p class="pt-2 pb-3">
+                            Pizza is a dish of Italian origin consisting of a usually round, flat base of leavened wheat-based dough topped with tomatoes, cheese, and often various other ingredients.
+                        </p>
+                    </div>
+                    <div class="col-md program_item">
+                        <i class="fa fa-lemon-o fa-2x pb-2 pt-3" aria-hidden="true"></i>
+                        <h4 class="py-2">Nems</h4>
+                        <p class="pt-2 pb-3">
+                            Nem cuốn (Gỏi cuốn in southern Vietnam), nem rolls, salad rolls, or summer rolls, is a Vietnamese dish traditionally consisting of pork, prawns, vegetables
+                        </p>
+                    </div>
+                    <div class="col-md program_item">
+                        <i class="fa fa-apple fa-2x pb-2 pt-3" aria-hidden="true"></i>
+                        <h4 class="py-2">Cake</h4>
+                        <p class="pt-2 pb-3">
+                            Cake is a flour confection made from flour, sugar, and other ingredients, and is usually baked. In their oldest forms, cakes were modifications of bread.
+                        </p>
+                    </div>
+                    <div class="col-md program_item">
+                        <i class="fa fa-cutlery fa-2x pb-2 pt-3" aria-hidden="true"></i>
+                        <h4 class="py-2">Milkshake</h4>
+                        <p class="pt-2 pb-3">
+                            A milkshake is a sweet beverage made by blending milk, ice cream, and flavorings or sweeteners such as butterscotch, caramel sauce, chocolate syrup.
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <a class="program_link">SEE THE WHOLE MENU</a>
+            <button class="btn btn-danger btn-lg">Join Délices Abidjanais</button>
+        </section>
+        <section class="cooks" id="contact">
+            <h3>Our chefs</h3>
+            <hr class="line mb-5">
+            <div class="cooks_container" id="cooks_container">
+
+            </div>
```

Comment:
- [Optional]  Dear @SFarida, I see you make changes to your chiefs that are not reflected on the live version of your project. Please make sure to have your fresh changes on the live server. as that would be presented to users.

Reply:
Oh thank you, I will do that